functions/Restore-DbaFromDatabaseSnapshot.ps1

```diff
@@ -0,0 +1,291 @@
+﻿Function Restore-DbaFromDatabaseSnapshot
+{
+<#
+.SYNOPSIS
+Restores databases from snapshots
+
+.DESCRIPTION
+Restores the database from the snapshot, discarding every modification made to the database
+NB: Restoring to a snapshot will result in every other snapshot of the same database to be dropped
+It also fixes some long-standing bugs in SQL Server when restoring from snapshots
+
+.PARAMETER SqlInstance
+The SQL Server that you're connecting to
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER Databases
+Restores from the last snapshot databases with this names only
+
+.PARAMETER Snapshots
+Restores databases from snapshots with this names only. NB: it overrides the -Databases parameter
```

Comment:
Here it says snapshots parameter overrides databases but in the code there's a termination when both exist.